bot/src/main.rs

```diff
@@ -196,6 +196,15 @@ struct Config {
 
     dry_run: bool,
 
+    /// compute score foll all validators in the cluster
+    score_all: bool,
```

Comment:
How about rebasing this PR on https://github.com/solana-labs/stake-o-matic/pull/118, where we add a `Command` enum to avoid adding command-specific (`ScoreAll` in this case) parameters into the `Config` struct